### DISCUSSION TITLE
How do I use `Tab` inside the editor

### DISCUSSION THREAD

**Discussion by whokilleddb** [nicknisi/dotfiles#231] [Q&A]
When I press the Tab key inside the editor, it does not seem to register and I have to manually type in the spaces. Any Idea what I am doing wrong??
<img width="659" alt="Screenshot 2024-09-26 at 6 18 58 PM" src="https://github.com/user-attachments/assets/d6070fa8-6cac-4329-99fb-1286026416f5">

(On the line after `printf`, I keep hitting the tab button but it does not seem to be doing its thing. Any idea how I can get around this?)

**Comment by nicknisi** [comment]
Haha whoa, I've never noticed that tab didn't work! I guess I've just trained myself not to use it. I'll look into it but in the mean time, you can use `>>` (shift + > twice) to indent the line.
![capture-20240926083047](https://github.com/user-attachments/assets/69464cba-1c46-4aeb-8227-75b7553e4859)
